README.md

```diff
@@ -11,19 +11,19 @@ To run this tool locally you need:
 
 ## Running locally
 
-* Restore the dotnet tools:
+* Perform a one-time set up of source dependencies:
 
-> dotnet tool restore
-
-* Restore the npm packages:
-
-> yarn
+> dotnet fake run build.fsx -t "Fantomas-Git"
 
 * Run the Watch target with FAKE:
 
 > dotnet fake run build.fsx -t Watch
 
-Open http://localhost:9060
+NOTE: you may see some error output during this process on first run, but those errors should eventually resolve and subsequent runs work without error.
+
+* Open http://localhost:9060
+
+NOTE: you will need to disable CORS for the front-end to interact with the locally-hosted backend. For example, if using Firefox, you can do this using the CORS Everywhere extension.
```

Comment:
This bit me as well, so just mentioned it here. Didn't want to get too detailed because it seems like the instructions vary quite drastically depending on platform and browser.

Reply:
Hey, sorry for the slow response.
I'm a bit puzzled why CORS is a problem here, https://github.com/fsprojects/fantomas-tools/blob/301e12b7312d5348c871a4ac53f9b6e9b36c122f/build.fsx#L32-L33

should take care of that.

Reply:
Ah, interesting - I didn't even see that. Here's what made me think CORS needed to be disabled:

![image](https://user-images.githubusercontent.com/1901832/128579806-3deaa3eb-08d2-48b5-997a-d6d347186735.png)

But I just found that the CORS URL configured by the `Watch` target is HTTPS rather than HTTP. Changing to HTTP fixed the problem, so included that in this PR and removed the above note from the README.

Reply:
Great spot! Thanks